src/ResourceManager/Intune/Commands.Intune/ApiVersionHandler.cs

```diff
@@ -0,0 +1,95 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
@markcowl : Actually we took out the all the refs to this file. No one is referring to this file. I'll remove it if you it is unnessacary.